capitalize-and-filter.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable indent */
```

Comment:
We can help you update your eslint to work with Prettier so you don't have to disable it on each file. It's a quick settings tweak in your VScode.